### PR TITLE
Announce 4.92

### PR DESCRIPTION
**Pull request by fredemmott**

### CODE REVIEW

File: _posts/2020-01-12-hhvm-4.92.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+HHVM 4.92 is released! This release marks the end of support for 4.83;
+HHVM 4.84&ndash;4.91 remain supported, as do the 4.56 and 4.80 LTS releases.
+
+(Note: HHVM 4.85, 4.89 and 4.90 were skipped.)
+
+# Highlights
+
+- If a class A is a supertype of class/interface B via `require extends`, it
+  is now able to call its' own private methods on instances of B.
+- Corrected error line number for reified generic errors.
```
Comment (jjergus) [inline _posts/2020-01-12-hhvm-4.92.markdown:17]: Is this a breaking change (FIXMEs may need to be moved) or is this just a change in some secondary line number?

File: _posts/2020-01-12-hhvm-4.92.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- Replaced sometimes-confusing "expected dynamic, got nonnull" with clearer
```
Comment (jjergus) [inline _posts/2020-01-12-hhvm-4.92.markdown:18]: is it "got ?nonnull" (equivalent to mixed) or "got nonnull" (not equivalent)?

File: _posts/2020-01-12-hhvm-4.92.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- `apc_cache_info()` now correctly returns an `uncounted_entries` key instead of
+  `uncounted_entires`.
+- Calling methods on `dynamic` values requires that all values are coercable to
+  `dynamic`, and the return type will be `dynamic`.
+- it is now a syntax error to declare a function as `async` without an
+  `Awaitable` return type; previously this was separately checked by the type
+  checker and the bytecode emitter, and the typechecker did not detect invalid
+  async lambdas.
+
+# Future Changes
```
Comment (jjergus) [inline _posts/2020-01-12-hhvm-4.92.markdown:34]: remove empty section?

**Review by jjergus — APPROVED** [review]

File: _posts/2020-01-12-hhvm-4.92.markdown (hunk 2 above)
Comment (fredemmott) [reply _posts/2020-01-12-hhvm-4.92.markdown:18]: This is for situations where you have a mixed, check it's not-null, then try to use it as dynamic.

`nonnull` is technically more correct, but users seem to to generally still think of their values as `mixed` instead - or, they read this as implying that the value should be null.

**Pull request merged**